client/client.go

```diff
@@ -478,14 +478,21 @@ func (r *NotaryRepository) GetTargetByName(name string, roles ...string) (*Targe
 
 }
 
+// TargetSignedStruct is a struct that contains information about the Target and the list of signatures
```

Comment:
nit: can we change this description to

` TargetSignedStruct is a struct that contains a Target, the role it was found in, and the list of signatures for that role`